host/GRpc.go

```diff
@@ -0,0 +1,104 @@
+package host
+
+import (
+	core "authz/api/gen/v1"
+	"authz/app"
+	"authz/app/contracts"
+	"authz/app/controllers"
+	"context"
+	"errors"
+	"net"
+	"os"
+	"sync"
+
+	"github.com/golang/glog"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/metadata"
+)
+
+// GRpc represents a GRpc host service
+type GRpc struct {
```

Comment:
is the uppercase GR here and in the file etc. intended? if so, why? also we should think about naming a bit i think. would love to do a refactoring collab, just ping me

Reply:
upper case: usually means it is "exposed" . I am not sure if that is intended here. but filenames in go, the best practice is to use lower case to start (refer packagenames section in: https://go.dev/doc/effective_go#package-names)

Reply:
then it should be `Grpc` not `GRpc` right? (and yes, I know that ;) )

Reply:
It was Grpc originally. The linter said to change it. >.>